### PR TITLE
Add Midas maker

### PR DESCRIPTION
**Pull request by MidasWireless**
Pull request for adding Midas full hotspot in Helium App.

### CODE REVIEW

**Comment by cokes518** [comment]
please update this PR with your maker ID and we'll merge it in.

**Comment by MidasWireless** [comment]
Already updated maker id. Please let me know if you can merge it in.

File: src/makers/midas/antennas.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { MakerAntenna } from '../antennaMakerTypes'
+
+const MIDAS_GATEWAY_US = {
+  name: 'MIDAS_GATEWAY(US 915)',
```
Comment (cokes518) [inline src/makers/midas/antennas.ts:4]: @MidasWireless do you want your Hotspot to appear in a list with an underscore in the name?

File: src/makers/midas/hotspots.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Midas to identify issues with your Hotspot in a secure way.</white></b>\n\nMidas will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@Midaswireless.com</b></purple> from the email used to purchase the Hotspot.',
+      power: [
+        'Plug in the provided power adapter into an outlet near a window and screw in the provided antenna on the back of the Hotspot.',
+        'The Hotspot is ready when the second indicator light is steady green.',
+      ],
+      bluetooth: [
+        'Use the provided pin to press the BT Button on the back of the Hotspot and hold for more than 15 seconds.',
+        'The hotspot is ready to pair when the third indicator light turns blue.\n\nMake sure your phone\'s Bluetooth is turned on! And Move the phone close to the device to increase the success rate of device connection!',
```
Comment (tyler-whitman) [inline src/makers/midas/hotspots.ts:19]: Please fix the lint error:
```
  19:9  error  Replace `'The·hotspot·is·ready·to·pair·when·the·third·indicator·light·turns·blue.\n\nMake·sure·your·phone\'s·Bluetooth·is·turned·on!·And·Move·the·phone·close·to·the·device·to·increase·the·success·rate·of·device·connection!'` with `"The·hotspot·is·ready·to·pair·when·the·third·indicator·light·turns·blue.\n\nMake·sure·your·phone's·Bluetooth·is·turned·on!·And·Move·the·phone·close·to·the·device·to·increase·the·success·rate·of·device·connection!"`  prettier/prettier
```

File: src/makers/midas/index.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 26,
```
Comment (tyler-whitman) [inline src/makers/midas/index.ts:7]: Please fix the lint error:

```
  7:10  error  Delete `·`  prettier/prettier
```

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Please fix the lint errors and resolve the merge conflicts with main. You can check for lint errors by running `yarn lint` in the project root.

File: src/makers/midas/index.ts (hunk 3 above)
Comment (MidasWireless) [reply src/makers/midas/index.ts]: already deleted the space after "id: 26,"

File: src/makers/midas/hotspots.ts (hunk 2 above)
Comment (MidasWireless) [reply src/makers/midas/hotspots.ts]: resolved by using "" and removing /

File: src/makers/midas/antennas.ts (hunk 1 above)
Comment (MidasWireless) [reply src/makers/midas/antennas.ts]: already modified hotspot names in antennas.ts and hotspots.ts

**Review by tyler-whitman — APPROVED** [review]
Thank you for the changes, looks good now.

**Pull request merged**